evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -115,10 +115,12 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'To CSV file' %}</h6>
                         <p class="card-text">{% trans 'This will create a CSV file containing login keys for all external participants.' %}</p>
                     </div>
                     <div class="card-footer text-center">
-                        <button name="operation" value="export-login-keys" type="submit" class="btn btn-sm btn-primary form-submit-btn"
-                        {% if not evaluation.has_external_participant %} disabled data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans "This evaluation has no external participants." %}"{% endif %}>
-                            {% trans 'Export login keys' %}
-                        </button>
+                        <div{% if not evaluation.has_external_participant %} data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans "This evaluation has no external participants." %}"{% endif %}>
+                            <button name="operation" value="export-login-keys" type="submit" class="btn btn-sm btn-primary form-submit-btn"
```

Comment:
Feels to me like this is worse HTML because the tooltip semantically belongs to the button, doesn't it? Why is the additional `div` necessary?

Reply:
Tooltips can't be activated on disabled elements.